tests-behat/bootstrap/Context.php

```diff
@@ -419,6 +430,139 @@ public function iClickIconUsingCss($arg1)
         $icon->click();
     }
 
+    /**
+     * Generic ScopeBuilder rule with select operator and input value.
+     *
+     * @Then /^rule "([^"]*)" operator is "([^"]*)" and value is "([^"]*)"$/
+     */
+    public function scopeBuilderRule($name, $operator, $value)
+    {
+        $rule = $this->assertScopeBuilderRuleExist($name);
+        $this->assertSelectedValue($rule, $operator, '.vqb-rule-operator select');
+        $this->assertInputValue($rule, $value);
+    }
+
+    /**
+     * hasOne reference or enum type rule for ScopeBuilder.
+     *
+     * @Then /^reference rule "([^"]*)" operator is "([^"]*)" and value is "([^"]*)"$/
+     */
+    public function scopeBuilderReferenceRule($name, $operator, $value)
+    {
+        $rule = $this->assertScopeBuilderRuleExist($name);
+        $this->assertSelectedValue($rule, $operator, '.vqb-rule-operator select');
+        $this->assertSelectedValue($rule, $value, '.vqb-rule-input select');
+    }
+
+    /**
+     * Date, Time or Datetime rule for ScopeBuilder.
+     *
+     * @Then /^date rule "([^"]*)" operator is "([^"]*)" and value is "([^"]*)"$/
+     */
+    public function scopeBuilderDateRule($name, $operator, $value)
+    {
+        $rule = $this->assertScopeBuilderRuleExist($name);
+        $this->assertSelectedValue($rule, $operator, '.vqb-rule-operator select');
+        $this->assertInputValue($rule, $value, 'input.form-control');
+    }
+
+    /**
+     * Boolean type rule for ScopeBuilder.
+     *
+     * @Then /^bool rule "([^"]*)" has value "([^"]*)"$/
+     */
+    public function scopeBuilderBoolRule($name, $value)
+    {
+        $this->assertScopeBuilderRuleExist($name);
+        $idx = ($value === 'Yes') ? 0 : 1;
+        $isChecked = $this->getSession()->evaluateScript('return $(\'[data-name="' . $name . '"]\').find(\'input\')[' . $idx . '].checked');
+        if (!$isChecked) {
+            throw new \Exception('Radio value selected is not: ' . $value);
+        }
+    }
+
+    /**
+     * @Then /^I check if word from data scope match$/
```

Comment:
The same here, just add the expected text to feature.

Reply:
or add a button into the test case that will do that comparison and check if the values are the same there

Reply:
I added it to the demo file because we can change it without changing the Context file.
Will try to set up using the argument.